docusaurus/docs/create-a-plugin/develop-a-plugin/good-practices.md

```diff
@@ -0,0 +1,66 @@
+---
+id: good-practices
+title: Good practices
```

Comment:
Maybe we could use best practices rather than good? 🤔

Reply:
Okay, probably best practices is a better naming